### PR TITLE
Bug 1823457, added AWS Private Link information

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1823457

@pweil- @sdodson PTAL

### CODE REVIEW

File: modules/installation-about-custom-gcp-vpc.adoc [hunk 1]
```diff
@@ -32,6 +32,15 @@ To ensure that the subnets that you provide are suitable, the installation progr
 * The subnet CIDRs belong to the machine CIDR.
 * You must provide a subnet to deploy the cluster control plane and compute machines to. You can use the same subnet for both machine types.
 
+If you are working in a disconnected environment, you are unable to reach the
+public IPs for EC2 and ELB endpoints. To resolve this, you must create a VPC
+endpoint and attach it to the subnet that the clusters are using. The endpoints
+should be named as follows:
+
+* `ec2.<region>.amazonaws.com`
```
Comment (pweil-) [inline modules/installation-about-custom-gcp-vpc.adoc:40]: this wouldn't apply to GCP, it is AWS specific.  I'm not aware of how it works in GCP, maybe @sdodson can help us out.

Comment (sdodson) [reply modules/installation-about-custom-gcp-vpc.adoc:40]: Yeah, this is not necessary for GCP. Lets drop this change.

**Comment by sdodson** [comment]
Changes to AWS doc look good to me, need to drop the GCP bit though.

File: modules/installation-about-custom-gcp-vpc.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/installation-about-custom-gcp-vpc.adoc]: @pweil- @sdodson Thanks! Updated.

**Comment by ahardin-rh** [comment]
@yunjiang29 Can you please provide QE review?

File: modules/installation-custom-aws-vpc.adoc [hunk 2]
```diff
@@ -33,6 +33,15 @@ Your VPC must meet the following characteristics:
 
 If you use a cluster with public access, you must create a public and a private subnet for each availability zone that your cluster uses. The installation program modifies your subnets to add the `kubernetes.io/cluster/.*: shared` tag, so your subnets must have at least one free tag slot available for it. Review the current link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions[Tag Restrictions] in the AWS documentation to ensure that the installation program can add a tag to each subnet that you specify.
 
+If you are working in a disconnected environment, you are unable to reach the
+public IPs for EC2 and ELB endpoints. To resolve this, you must create a VPC
```
Comment (jboxman) [inline modules/installation-custom-aws-vpc.adoc:37]: In past reviews, I've been requested to s/IPs/IP addresses/.

File: modules/installation-custom-aws-vpc.adoc [hunk 3]
```diff
@@ -33,6 +33,15 @@ Your VPC must meet the following characteristics:
...
+endpoint and attach it to the subnet that the clusters are using. The endpoints
+should be named as follows:
+
+* `ec2.<region>.amazonaws.com`
```
Comment (jboxman) [inline modules/installation-custom-aws-vpc.adoc:41]: Do we want to specify what to provide for `<region>`?

**Review by jboxman — APPROVED** [review]
Two items but otherwise LGTM.

File: modules/installation-custom-aws-vpc.adoc (hunk 3 above)
Comment (ahardin-rh) [reply modules/installation-custom-aws-vpc.adoc:41]: @pweil- @sdodson Should we list options here? Thanks!

**Review by yunjiang29 — APPROVED** [review]
LGTM

File: modules/installation-custom-aws-vpc.adoc (hunk 3 above)
Comment (sdodson) [reply modules/installation-custom-aws-vpc.adoc:41]: The list of regions will change over time and I'd expect an AWS admin to be able to extrapolate from what you have already.

Comment (pweil-) [reply modules/installation-custom-aws-vpc.adoc:41]: > The list of regions will change over time and I'd expect an AWS admin to be able to extrapolate from what you have already.

+1.  If we want to do anything I'd just link to Amazon documentation and not try to maintain the canonical list here.

**Pull request merged**